README.md

```diff
@@ -51,7 +51,7 @@ Where you need to create crontab.txt and your borgmatic config.yml
 - To generate an example borgmatic configuration, run:
 ```
 docker exec borgmatic \
-sh -c "generate-borgmatic-config -d /etc/borgmatic.d/config.yaml"
+sh -c "cd && generate-borgmatic-config -d /etc/borgmatic.d/config.yaml"
```

Comment:
Why does this start with `cd` ?

Reply:
To change the workdir to the user home, otherwise the command will not work.
Alternative is to use `--workdir`, but I had in mind that #27 will be implemented one day.

Reply:
Fair point!